examples/gin/main.go

```diff
@@ -0,0 +1,136 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/airbrake/gobrake/v5"
+	ginbrake "github.com/airbrake/gobrake/v5/gin"
+	"github.com/gin-gonic/gin"
+)
+
+var ProjectId int64 = 105138                               // Replace it with your project Id
```

Comment:
Nitpick: the comment on this line is not the same as the comment in `examples/apexlog/main.go` line 12. I would make it the same just for consistency.